conf/modules.config

```diff
@@ -275,7 +275,10 @@ process {
         ext.args2        = { params.use_gatk_spark && params.use_gatk_spark.contains('markduplicates') && (!params.skip_tools || (params.skip_tools && !params.skip_tools.contains('markduplicates'))) ? '-n' : '' }
         ext.prefix       = { params.split_fastq > 1 ? "${meta.id}".concat('.').concat(reads.get(0).name.findAll(/part_([0-9]+)?/).last()) : "" }
         publishDir       = [
-            enabled: false
+            enabled: (params.skip_tools && params.skip_tools.contains('markduplicates')),
+            mode: params.publish_dir_mode,
+            path: { "${params.outdir}/preprocessing/${meta.sample}/mapped" },
```

Comment:
Same here.
And it'll be good to actually fix meta.id with intervals when no intervals.
I had to use `meta.sample` here instead of `meta.id`

Reply:
on it with #524